in-progress/roles/mtbc.nginx-configure/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: configure nginx - install OMERO.web configuration
```

Comment:
What's the plan for getting rid of "in-progress" and "mtbc" in these file paths?

Reply:
I don't have one but I like to imagine that eventually me or somebody else might get to come back around to making these more as we'd want them to be or they may become superfluous because of work on other things. It may be that there are better devices to separate them out as being a bit rubbish even though they're already useful for managing those production servers, happy to adjust accordingly.

Reply:
(E.g., I think`mtbc.omero-web-apps` might disappear if `omero-web` gains an "include these apps" setting.)

Reply:
Considering this is our main gallery of playbooks, I'd prefer us _not_ to have "in-progress" and our personal accounts showing up. If the roles aren't covered by anything else, then the best course of action is likely to get them on galaxy.

Reply:
I could also just put them somewhere on squig if that helps, I just wanted them team-visible really.

Reply:
That would make these playbooks unconsumable by anyone else which goes again the idea of having them in `prod-playbooks`.

Reply:
Oh! You mean, put the playbooks just among the others and somehow put the mtbc roles with their current names on galaxy?

Reply:
Yup. Just `s/mtbc/ome/`. That might require some quick review to see if there are any dupes.

Reply:
Will move the roles to be `ome.*` ones in my GitHub repo for review / adoption.

Reply:
Now moved to:
- https://github.com/mtbc/ansible-role-ldap-cert
- https://github.com/mtbc/ansible-role-nginx-configure
- https://github.com/mtbc/ansible-role-omero-web-apps